yoti-sdk-api/src/main/java/com/yoti/api/client/Profile.java

```diff
@@ -1,44 +1,55 @@
 package com.yoti.api.client;
 
 import java.util.Collection;
+import java.util.List;
 
 /**
  * Yoti profile for a connect token and application. You can get a hold of one of these by using a {@link YotiClient}.
  */
 public interface Profile {
 
     /**
-     * Return typed attribute value for a key.
+     * Return typed {@link Attribute} object for a key.
      *
      * @param <T> the type parameter indicating the type of the returned value
      * @param name attribute name
      * @param clazz attribute type
-     * @return typed attribute, null if attribute type is not assignable from the specified class
+     * @return typed attribute, null if it is not present in the profile
      */
     <T> Attribute<T> getAttribute(String name, Class<T> clazz);
 
     /**
-     * Returns the attribute object for the key
+     * Returns the {@link Attribute} object for the key
      *
      * @param name
      * @return the attribute object, null if it is not present in the profile
      */
     Attribute getAttribute(String name);
 
     /**
-     * Return the value of the first attribute with a name starting with <code>name</code>
+     * Return a list of all the {@link Attribute} with a name starting with <code>name</code>
```

Comment:
Does it work if we add an `s` here so it's `a list of all the {@link Attribute}s`?

Reply:
Seems to display ok